packages/snaps-controllers/src/snaps/registry/json.test.ts

```diff
@@ -372,6 +453,21 @@ describe('JsonSnapsRegistry', () => {
       expect(result).toBe('1.0.0');
     });
 
+    it('resolves to a compatible allowlisted version', async () => {
```

Comment:
Can you add a test to make sure we resolve to `1.1.0` if the client version allows for it?